test/library/draft/cuBLAS/level1/test_cublas1.skipif

```diff
@@ -0,0 +1,4 @@
+# TODO this should be based off whether we can find `nvcc` or on Cray-X* a `cc`
+# wrapper that's similar to `nvcc`. However, I'm not sure how we detect if
+# we're going to launch onto a compute node that has GPUS
```

Comment:
You mean this as an additional check here, right? I think there should be a `CHPL_TEST_GPU`-like environment and we should check against that even if we can find a suitable compiler.

Reply:
Right, I think there's 2 components. Can we compile for GPUs, and are we going to run on nodes with a GPU. Currently this env var is a proxy for both. Ideally I'd like to automate the check for both components, but I'm not sure how practical it is to automate the check that we're going to run on nodes with a GPU.